ethereum/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM ethereum/client-go:stable
+
+RUN apk add --no-cache curl
+
+COPY tests/assets/ETHGenesis.json ETHGenesis.json
+
+RUN geth --identity "PeggyTestnet" \
+    --nodiscover \
+    --networkid 15 init ETHGenesis.json
+
+ENTRYPOINT geth --identity "PeggyTestnet" --nodiscover \
+           --networkid 15 \
+           --mine \
+           --http \
+           --http.port "8545" \
+           --http.addr "0.0.0.0" \
+           --http.corsdomain "*" \
+           --http.vhosts "*" \
+           --miner.threads=1 \
+           --nousb \
+           --verbosity=5 \
+           --miner.etherbase=0xBf660843528035a5A4921534E156a27e64B231fE
```

Comment:
This is a fun magic number.

Reply:
I got it from here: https://github.com/cosmos/gravity-bridge/blob/fd98d205b8db4575112b29735d77ecd569d67a1b/solidity/scripts/contract-deployer.sh#L6

If it doesn't have purpose, I will remove it. But it was there before